qiskit/visualization/gate_map.py

```diff
@@ -114,6 +116,14 @@ def plot_gate_map(backend, figsize=None,
     n_qubits = config.n_qubits
     cmap = config.coupling_map
 
+    if qubit_labels is None:
+        qubit_labels = list(range(n_qubits))
+    else:
+        if len(qubit_labels) != n_qubits:
+            raise QiskitError('Length of qubit labels '
```

Comment:
I would probably use a `VisualizationError` for this

Reply:
Done.